docs/sources/developers/plugins/publish-a-plugin.md

```diff
@@ -9,7 +9,8 @@ type: docs
 
 # Publish a plugin
 
-You've just built your first plugin, and now you want to share it with the world.
+You've just built your first plugin; now you want to share it with the world.
```

Comment:
Not a fan of "You've built your fist plugin" as implies this is only beginner content rather than something to refer back to

Reply:
Fixed in latest commit.